<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maintenance</class>
 <widget class="QMainWindow" name="Maintenance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maintenance</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>POSSIBLE TEAM TO ADD:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="addTeamCombo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>San Diego Sailors</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="addTeamButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Click To Add Team</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify Team Souvenirs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Please Select the Team You Wang To Edit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="teamSelectSouveirCombo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Please Select the Souvenir You Want To Modify</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectSouvenirCombo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Please Input The New Price Here:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changePriceButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Click To Change Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete This Souvenir?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteSouvenirButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>160</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Click To Delete</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="newSouvenirNameInput">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>140</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Please Enter The New Name Of The New Souvenir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Please Input The New Price Here:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSouvenirButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Click To Add Souvenir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify Team Informations</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify Team Stadium:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="newStadiumNameInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify Team Capacity:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="newLocationNameInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify Team Location:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify Team Roof Type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="modifyRoofTypeCombo">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>480</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Retractable</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Open</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fixed</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="newSurfaceNameInput">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>560</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>520</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify Surface:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>610</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify Year Opened:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="modifyConferenceCombo">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>480</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>National Football Conference</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>American Football Conference</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>440</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify Conference:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="modifyDivisionCombo">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>650</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>NFC West</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NFC North</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NFC South</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NFC East</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AFC West</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AFC North</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AFC South</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AFC East</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>610</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify Team Division:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeTeamInfoButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>530</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Click To Modify Team</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="souvenirPriceInput">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="newYearOpenedInput">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>650</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="newSouvenirPriceInput">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="newTeamCapacityInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1297</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>teamSelectSouveirCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>selectSouvenirCombo</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>teamSelectSouveirCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>selectSouvenirCombo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectSouvenirCombo</sender>
   <signal>activated(int)</signal>
   <receiver>Maintenance</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
